src/app/containers/ATIAnalytics/params/frontpage/index.test.jsx

```diff
@@ -54,60 +63,13 @@ describe('FrontPageAtiParams', () => {
     });
 
     it('should call atiPageViewParams with the params from the Contexts', () => {
-      const mock = jest.fn().mockReturnValue('key=value&key2=value2');
-      atiPageViewParams.default = mock;
-
-      renderer.create(Component(serviceContextStub, requestContextStub));
-
-      expect(mock).toHaveBeenCalledTimes(1);
-      expect(mock).toHaveBeenCalledWith({
```

Comment:
Wheres this gone?

Reply:
https://github.com/bbc/simorgh/pull/3559/files#diff-969e14df75b9da2e46ff6540eabc15a0

Since the `buildFrontPageATIUrl` is already tested, we should only test that it's called, and called with the right arguments.